docs/oauth.md

```diff
@@ -211,6 +217,19 @@ tsm configuration set -k oauth.config.clients -v "[{\"oauth.config.id\":\"[your_
 Replace [your_dbclass] with the `dbclass` element registered in your oauthConfig.xml file. Substitute [your_client_id], [your_client_secret], [your_redirect_url] with the ones you registered in your provider's OAuth registration page.
 [your_redirect_url] needs to follow certain format, if your server address is https://Myserver/ then [your_reirect_uri] needs to be https://Myserver/auth/add_oauth_token.
 
+Tableau Online is managed by Tableau, in order for you connector to work on Tableau Online you will need to provide us(instruct us on how to create) the set of clientId/clientSecret/redirect_uri. 
+
+## Site-Wide OAuth Clients
+Another way is through the site level OAuth client feature where the server admin for Tableau Server/ site admin for Tableau Online will be able to register the oauth client on a particular site, for example setting Azure AD oauth client on a site: https://help.tableau.com/current/server/en-us/config_oauth_azure_ad.htm.  
+The oauth clients will only be effective in the particular site, it did not require a restart and take precedence over the server wide oauth clients if any.
+
+| Server-Wide OAuth Clients  | Site-Wide OAuth Clients | 
+| ----  | ------- | 
+| tsm command running by server admin | User Interface and can be accessed by site admin(Tableau Online) or server admin(Tableau Server)|
+| Need restart server| No need to restart server|
+| Apply to whole server| Only apply to the particular site, will not affect other sites|
+| Low priority | High priority |
```

Comment:
I don't understand this line.

Reply:
This is saying that the Site Level OAuth client will override the Server Level OAuth Client settings.

Reply:
I've changed the wordings there, let me know if it's better.